src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Application.cshtml

```diff
@@ -6,6 +6,17 @@
 @{
     ViewBag.Title = "Gateway evaluation";
     Layout = "_Layout";
+
```

Comment:
Might be better to have this logic outside the view so it can be unit tested

Reply:
We need to remove the logic altogether.